frontend/src/components/Player.vue

```diff
@@ -3,7 +3,10 @@
     <div class="info">
       <div class="name-info">
         <!-- <div class="avatar"><img src="../assets/icons/..." alt="avatar"></div> -->
-        <div class="name title-font">{{ player.name }}</div>
+        <div class="name title-font">
+          {{ player.name }}
+          <AwardIcon v-if="winner" :title="$t('badge_description', {name: player.name})" size="18" />
```

Comment:
joke's on me. The `title` attribute doesn't work as intended here because the resulting code is going to be an `<svg>` element and `title` attributes don't work properly there. Nothing to do here, we can keep this as is. But I think I'll look into a `popover` or `tooltip` kind of component pretty soon since `title` is in fact problematic for accessibility as I learned the other day :)

Reply:
I'm no Frontend expert, but I was wondering why I didn't notice a change 😅
Nethertheless, it's content in html and therefore adds value (I'm thinking of SEO, although I see no use-case here 😏)

Totally agree on tooltip components, this is why I suggested this earlier.
I'm excited to find out what you will come up with 👍